src/_shims/node-runtime.ts

```diff
@@ -40,11 +38,11 @@ async function fileFromPath(path: string, ...args: any[]): Promise<File> {
   return await _fileFromPath(path, ...args);
 }
 
-const defaultHttpAgent: Agent = new KeepAliveAgent({ keepAlive: true, timeout: 5 * 60 * 1000 });
-const defaultHttpsAgent: Agent = new KeepAliveAgent.HttpsAgent({ keepAlive: true, timeout: 5 * 60 * 1000 });
+const defaultHttpAgent = new uf.Agent({ keepAliveTimeout: 5 * 60 * 1000 });
+const defaultHttpsAgent = new uf.Agent({ keepAliveTimeout: 5 * 60 * 1000 });
```

Comment:
Is this necessary to get reasonable `keepAlive` behavior? IIRC, `node-fetch` didn't do keepAlive by default but if undici does, maybe we don't need to construct a default agent after all?

(note we also have weird timeout shenanigans elsewhere to bump the agent's timeout so that a long timeout for a given request isn't cut short by the agent's timeout; not sure whether that'd still be necessary or would need adjustment.)

Reply:
I'd have to go look at the default agent timeout to be sure

Reply:
FYI: undici will honor the keep-alive hint from the server.